### PR TITLE
added login

### PR DESCRIPTION
**Pull request by lavanya-thota**

### CODE REVIEW

**Comment by lavanya-thota** [comment]
added login just check and review

File: product-management/src/App.jsx [hunk 1]
```diff
@@ -1,11 +1,12 @@
 import {BrowserRouter, Route, Routes } from "react-router-dom";
 import { Navbar } from './navbar';
-import { Login } from './login';
+import  Login  from './login/Login';
```
Comment (NageshGandikota549) [inline product-management/src/App.jsx]: you can remove /Login at the end

File: product-management/src/login/Login.jsx [hunk 2]
```diff
@@ -1,5 +1,54 @@
...
 
-export const Login = () => {
-  return <div>Login Details</div>;
-};
+function Login() {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  const handleUsernameChange = (event) => {
+    setUsername(event.target.value);
```
Comment (NageshGandikota549) [inline product-management/src/login/Login.jsx:8]: pass the event.target.value as a param like below

const handleUsernameChange = (value) => {
    setUsername(evalue);
    }

File: product-management/src/navbar/Navbar.jsx [hunk 3]
```diff
@@ -5,7 +5,7 @@ import './navbar.css';
 export const Navbar = () => {
 return <div className="navbar-conatiner">       
   <div className="item"><Link to={'/'}>Home</Link></div>
-  <div className="item"><Link to={'/login'}>Login</Link></div>
+  <div className="item"><Link to={'/Login'}>Login</Link></div>
```
Comment (NageshGandikota549) [inline product-management/src/navbar/Navbar.jsx]: why the path changed to Uppercase ?

**Review by NageshGandikota549 — APPROVED** [review]

**Pull request merged**